jekyll/_docs/introduction.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: classic-docs
+title: Introduction to Continuous Integration and CircleCI
+short-title: Introduction to CircleCI
+categories: [getting-started]
+description: "An introduction to Continuous Integration, Continuous Deployment, and how CircleCI can help."
+---
+
+Typically CircleCI is used to do [Continuous Integration][wiki-ci] and Continuous Deployment. 
+
+## Continuous Integration 
+
+The general idea is to test all changes that you are making to your code base. You can accomplish this with: 
+
+* [Unit Tests][wiki-unittest]
+* [Integration Tests][wiki-inttest]
+* [Functional Tests][wiki-functest] (also known as End to End tests) 
+
+[Software Testing][wiki-codetest] in general is a huge topic with tons of concepts. The way that you write tests varies depending on what language and framework you are using.
+
+## Continuous Deployment 
+
+If your tests pass, then you can deploy your code to development, staging, production, etc... The specific way that you do this depends on what type of infrastructure you are deploying too. Some examples:
```

Comment:
If your tests pass, you can deploy your code to development, staging, production, or other environments. The way you do this will depend on the infrastructure you are deploying to. Some examples: